package.json

```diff
@@ -47,17 +50,18 @@
   "types": "./dist/types.d.ts",
   "type": "module",
   "files": [
-    "dist"
+    "dist",
+    "internals.d.ts"
   ],
   "scripts": {
     "prepare": "git config --local core.hooksPath .githooks",
-    "build": "nuxi build-module .",
-    "stub": "nuxi build-module --stub .",
+    "build": "nuxt-module-build build .",
```

Comment:
This script should be `nuxt-module-build prepare && nuxt-module-build build .`, you need to create `.nuxt` stuff before running other scripts